src/graph/graphcontext.h

```diff
@@ -291,15 +291,31 @@ int GraphContext_DeleteIndex
 // remove a single node from all indices that refer to it
 void GraphContext_DeleteNodeFromIndices
 (
-	GraphContext *gc,
-	Node *n
+	GraphContext *gc,  // graph context
+	Node *n,           // node to remove from index
+	LabelID *labels,   // [optional] node labels to remove from index
+	uint label_count   // [optional] number of labels
 );
 
 // remove a single edge from all indices that refer to it
 void GraphContext_DeleteEdgeFromIndices
 (
-	GraphContext *gc,
-	Edge *e
+	GraphContext *gc,  // graph context
+	Edge *e            // edge to remove from index
+);
+
+// add node to any relevant index
+void GraphContext_AddNodeToIndices
+(
+	GraphContext *gc,  // graph context
+	Node *n            // node to add to index
```

Comment:
Why `GraphContext_DeleteNodeFromIndices` requires the node's labels to be passed and `GraphContext_AddNodeToIndices` doesn't ?

Reply:
GraphContext_DeleteNodeFromIndices don't always delete the node from all indices
GraphContext_AddNodeToIndices always add it to all indices